syslog-ng-ctl/syslog-ng-ctl.c

```diff
@@ -191,19 +191,26 @@ slng_reload(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
 }
 
 static gboolean config_options_preprocessed = FALSE;
+static gboolean config_options_verify = FALSE;
 
 static GOptionEntry config_options[] =
 {
   { "preprocessed", 'p', 0, G_OPTION_ARG_NONE, &config_options_preprocessed, "preprocessed", NULL },
+  { "verify", 'v', 0, G_OPTION_ARG_NONE, &config_options_verify, "verify", NULL },
   { NULL,           0,   0, G_OPTION_ARG_NONE, NULL,                         NULL,           NULL }
 };
 
+
 static gint
 slng_config(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
 {
-  return _dispatch_command(config_options_preprocessed ? "CONFIG PREPROCESSED" : "CONFIG ORIGINAL");
+  if (config_options_verify)
+    return _dispatch_command("CONFIG VERIFY");
```

Comment:
I think the `CONFIG ORIGIN/PREPROCESSED (VERIFY)` would be a better way to go, because if at some point somebody wants/could implement `CONFIG PREPROCESSED VERIFY` it would not break the API here.

or better break the previous one now, and introduce something like this:
`CONFIG <GET/VERIFY> <ORIGINAL/PREPROCESSED>` because I think this later on would be easier to add new options.

What do you think ?

Reply:
Does this look like what you're recommending? 
I think I can go through this now unless there's a blocker.
```
static gint
slng_config(int argc, char *argv[], const gchar *mode, GOptionContext *ctx)
{
  if (config_options_verify)
    return _dispatch_command(config_options_preprocessed ? "CONFIG VERIFY PREPROCESSED" : "CONFIG VERIFY ORIGINAL");
  else
    return _dispatch_command(config_options_preprocessed ? "CONFIG PREPROCESSED" : "CONFIG ORIGINAL");
}

```

Reply:
Also I don't understand, what would this compare exactly?
`` syslog-ng-ctl/syslog-ng-ctl config --verify --preprocessed ``

Reply:
@Kokan - Would you please advise on that? I want to make sure to update it if needed before committing.

Reply:
I traced it! - I think the path is also stored in memory at: ``self -> options -> preprocess_into``